app/models/application_search_filter.rb

```diff
@@ -13,10 +14,22 @@ class ApplicationSearchFilter < ApplicationStruct
   # If a user is specified, only applications assinged to that user are returned.
   #
   def assigned_status_options
-    [
-      [I18n.t('labels.assigned_status.unassigned'), 'unassigned'],
-      [I18n.t('labels.assigned_status.assigned'), 'assigned']
-    ] + assigned_to_user_options
+    status_options = Types::ASSIGNED_STATUSES.map do |status|
+      [I18n.t(status, scope: 'values.assigned_status'), status]
+    end
+
+    status_options + assigned_to_user_options
+  end
+
+  #
+  # Options for the application status filter
+  #
+  # Includes 'Open', 'Completed', 'Sent back to provider' or 'All applications'
+  # Values can be "open", "completed", "sent_back", "all"
+  def application_status_options
+    Types::REVIEW_APPLICATION_STATUSES.keys.map do |status|
+      [I18n.t(status, scope: 'values.application_status'), status]
```

Comment:
If we are mapping the values to keys for display purposes - why not just keep the values the same as in the schema and just display them differently in review? Why remap?